community_guidelines.md

```diff
@@ -0,0 +1,100 @@
+# Community Guidelines
```

Comment:
I think it's better to move this page to `/docs/source/guide/` and make it as a part of our docs. Also we should publish a link to this help somewhere near this Join community button:
![image](https://user-images.githubusercontent.com/501780/214444203-3c44228b-f285-43c9-a1d0-52d3084bba69.png)